fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/projects/damp-acw-together-again.md

attribution: DAMP & A Constructed World
images:
  - src: Untitled_10.jpg
    alt: Blue plastic tub filled with art materials including framed images, books with "REVOLUTION" text, and various art objects
  - src: image350.jpg
    alt: Collage artwork showing drawings of a dove, sleeping cat, and eels swimming, with "A FACE MELTING SOLO" text
  - src: DampACW.jpg
    alt: Dark gallery installation with blue-lit aquatic containers, black panels displaying white photographs, and a white sculptural chair
  - src: eels.jpg
    alt: Group of visitors gathered around large blue tub observing live eels during "Explaining Contemporary Art to Live Eels" performance
  - src: ACW.jpg
    alt: Two artists crouching beside blue tub with orange-gloved hand reaching into water containing live eels and art materials
  - src: Untitled_1.jpg

>     alt: Floor detail showing live eel, orange glove, illuminated colored objects, and papers during performance

Reviewer: ```suggestion
    alt: Floor detail showing orange gloved hand holding live eel next to art objects and papers during performance
```

  - src: Untitled_3.jpg
    alt: Two people in red and black clothing leaning over blue tub filled with water and artworks, examining contents
  - src: Untitled_13.jpg

>     alt: Close-up of hand wearing orange glove reaching into blue tub with several live eels swimming in water

Reviewer: ```suggestion
    alt: Close-up of hand wearing orange glove moving eel from a pot into a large blue tub with several live eels swimming in water
```

  - src: BandRoom.jpg
    alt: Small room bathed in red light, densely packed with musical instruments, Marshall amplifiers, guitars on walls, and illuminated TV monitor
  - src: DeadDamp.jpg

>     alt: White wall displaying framed group photograph mounted on black painted form with dripping effect and wooden stick protruding

Reviewer: ```suggestion
    alt: White wall displaying group photograph positioned on black painted frame with dripping effect spelling DAMP
```

  - src: Start.jpg
    alt: Array of hand-painted cardboard guitars and Marshall amplifier paintings arranged on floor beneath colorful string lights
  - src: Untitled_19.jpg
    alt: Close-up view of painted cardboard Marshall amplifiers and decorated cardboard guitars with various colorful patterns
  - src: Scott.jpg
    alt: Person in dark clothing standing among painted cardboard guitars and Marshall amplifier backdrop with string lights
  - src: Ed.jpg

>     alt: Shirtless performer wearing underwear, headband, and sneakers holding white cardboard guitar during air guitar competition

Reviewer: ```suggestion
    alt: Shirtless performer wearing underwear, headband, and sneakers holding white cardboard guitar in front of cardboard guitar and amp display
```
